client/src/components/ratings/sub-components/ReviewTile.jsx

```diff
@@ -1,22 +1,18 @@
 import React, { useState, useEffect } from 'react';
 import { AiFillCheckCircle } from 'react-icons/ai';
+import { RxDividerVertical } from 'react-icons/rx';
 import ReviewImageModal from './ReviewImageModal.jsx';
 import fetcher from '../../../fetchers';
+import StarRating from '../../../helpers/star-rating/starRating.jsx';
```

Comment:
```suggestion
import StarRating from '../../../helpers/star-rating/StarRating.jsx';
```
Needs correct file name

Reply:
Great catch, might explain the warning I got from web-pack too.